components/next_steps_view/steps/enable_notifications_step/enable_notifications_step.tsx

```diff
@@ -57,12 +59,12 @@ export default function EnableNotificationsStep(props: StepComponentProps) {
         return (
             <div>
                 <TextCardWithAction
-                    cardBodyMessageId={'next_steps_view.notificationSetupNotificationsDisabled'}
+                    cardBodyMessageId={t('next_steps_view.notificationSetupNotificationsDisabled')}
                     cardBodyDefaultMessage={
                         'Notifications were previously disabled or you may be browsing in private mode. You\'ll need to open your browser settings or turn off private mode to enable notifications.'
                     }
                     buttonMessageId={
-                        'next_steps_view.notificationSetup.skipThisStep'
+                        t('next_steps_view.notificationSetup.skipThisStep')
```

Comment:
Noticed that for `TextCardWithAction` usages, the ids weren't automatically picked up by `make i18n-extract`, so took the liberty of adding explicit markup for them to be included. Which means they are now added to `i18n/en.json`, too.

Reply:
Thanks!